docs/tutorial/part-three/index.md

```diff
@@ -165,19 +164,19 @@ export default () => (
 )
 ```
 
-Let's see what the new about page looks like:
+Vamos ver como a nova página *sobre* ficou:
```

Comment:
```suggestion
Vamos ver como a nova página _sobre_ ficou:
```
Creio que seja melhor usar o italico para evidenciar, no lugar do negrito

Reply:
Na verdade, os dois casos estão em itálico. Negríto é `**texto**`, itálico pode ser `_texto_` ou `*texto*`. Mas vou aceitar a sugestão pra manter no padrão do projeto.

Reply:
verdade, erro meu